src/components/login/BackButton.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
```

Comment:
does every button have to be a new .js file?

Reply:
No but it helps for portability and reusability